src/Raven.Server/Documents/PeriodicBackup/Retention/RetentionPolicyRunnerBase.cs

```diff
@@ -63,7 +63,7 @@ public void Execute()
                 while (hasMore)
                 {
                     var foldersResult = GetSortedFolders();
-                    var resultType = hasMore ? "partial " : string.Empty;
+                    var resultType = foldersResult.HasMore ? "partial " : string.Empty;
```

Comment:
@grisha-kotler @garayx is there anything to merge here? What about this change?

Reply:
I will push it here: https://github.com/ravendb/ravendb/pull/11800
And we can close this PR